templates/default/pyenv.sh.erb

```diff
@@ -1,12 +1,6 @@
 # Prefer a user pyenv over a system wide install
```

Comment:
do we still want to create this file?

Reply:
The thought was that it wouldn't hurt and it's an easy way to ease the use of pyenv for a user, but now I'm reconsidering because the purpose of this change was to make it more clear that we're not responsible for supporting a user's pyenv. I'll make this change.